### DISCUSSION TITLE
Performance profiles

### DISCUSSION THREAD

**Discussion by Akryum** [vuejs/devtools-v6#1968] [General]
Share your devtools performance profiles in this thread!

[How to guide](https://devtools.vuejs.org/guide/devtools-perf.html)

**Comment by imaverickk** [comment]
System:
    OS: Windows 10 10.0.19043
Browsers:
    Chrome: 107.0.5304.122

Vue Devtools version: 6.4.5

What was done: unmount 199 components, then create 199 components

[Profile-20221202T002816.zip](https://github.com/vuejs/devtools/files/10134205/Profile-20221202T002816.zip)

**Comment by n2ai** [comment]
Is there any way that we can retain the rendering performances while using Vue devtools as I ran into a promise error/delay everytime using it.
